src/generate/apiGroup.ts

```diff
@@ -5,19 +5,18 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import { Api } from "./api";
-import Name from "./name";
+import { Name } from "./name";
 
 /**
  * A group of API objects. Common transformations of the group name are cached
  * for reference in templates and file paths.
  */
-export class ApiGroup {
-  apis: Api[];
+export class ApiGroup extends Array<Api> {
   name: Name;
 
-  constructor(name = "", apis: Api[] = []) {
+  constructor(name = "", ...apis: Api[]) {
```

Comment:
```suggestion
  constructor(name = "", apis: Api[] = []) {
```
I think it makes more sense for the arguments to be "a name and an array of all the APIs you want" rather than "all the APIs you want, except the first one is a name".